docs/src/use/configure/configuration-files.md

```diff
@@ -495,3 +498,152 @@ npx eslint --config some-other-file.js **/*.js
 ```
 
 In this case, ESLint does not search for `eslint.config.js` and instead uses `some-other-file.js`.
+
+<a name="experimental-typescript-support">
+
+## ◆ Configuration Files written in TypeScript (new)
+
+</a>
```

Comment:
Heading are automatically turned into links, and we don't use the diamond in any other headings.
```suggestion
## TypeScript Configuration Files
```